### DISCUSSION TITLE
다양한 팁이나 스킬

### DISCUSSION THREAD

**Discussion by protocol5** [protocol5/2021ESE-Mooyaho_team#6] [Ideas]
## 다양한 팁 or 스킬
리눅스 명령어나 활용도 좋은 응용프로그램, 익스텐션등
본인이 괜찮다고 생각하는 정보를 자유롭게 올려주면 됩니다.

**Comment by protocol5** [comment]
버추얼 박스 공유폴더 설정 시 마운트 폴더의 권한 문제가 생길 수 있음.
$ sudo adduser $USER vboxsf 
해당 명령어를 통해 vboxsf그룹에 유저를 추가해야함

**Comment by protocol5** [comment]
scp를 활용한 파일 폴더 전송
https://daeunginfo.blogspot.com/2019/04/scp.html

**Comment by DongJune96** [comment]
VNC viewer를 통해 라즈베리파이 화면이 원격으로 출력이 되지 않을 때(hdmi를 연결하지 않으면 화면이 출력되지 않는 이슈). 
윈도우10에 기본으로 내장되어있는 '원격 데스크톱 연결'을 사용하면 쉽게 화면 출력 가능
